core/src/test/java/org/bitcoinj/utils/FiatTest.java

```diff
@@ -84,6 +83,7 @@ public void testToPlainString() {
     public void testComparing() {
         assertTrue(parseFiat("EUR", "1.11").isLessThan(parseFiat("EUR", "6.66")));
         assertTrue(parseFiat("EUR", "6.66").isGreaterThan(parseFiat("EUR", "2.56")));
+        assertEquals(parseFiat("EUR", "0.80").compareTo(parseFiat("USD","0.80")), -16);
```

Comment:
With `assertEquals()`, the expected value comes first. Assuming you exchanged the who arguments and `-16` is the expected value: why do you expect -16?